content/docs/operating/integrations.md

```diff
@@ -91,6 +91,7 @@ For notification mechanisms not natively supported by the Alertmanager, the
   * [Matrix](https://github.com/matrix-org/go-neb)
   * [Phabricator / Maniphest](https://github.com/knyar/phalerts)
   * [prom2teams](https://github.com/idealista/prom2teams): forwards notifications to Microsoft Teams
+  * [prom2tower](https://github.com/pja237/prom2tower): call Ansible Tower (AWX) API on alerts (launch jobs etc.)
```

Comment:
Most items in this list are not named by the name of the tool, but by the thing they integrate with. The existing `prom2teams` right above is an exception and maybe should be changes to just `Microsoft Teams` as well. I think for this item, just `Ansible Tower` would be the expected item name?

Reply:
Sure, no problem with that, would you like me to push a change with "Ansible Tower" then?

Reply:
Yep please :)

Reply:
Done! :)